tests/testapps/CTCatalog/CTCatalog.vsimporter/CTCatalog-WinStore10/CTCatalog.vcxproj

```diff
@@ -233,7 +233,11 @@
       <PrecompiledHeader>Create</PrecompiledHeader>
     </ClCompile>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Font Include="..\..\CTCatalog\Fonts\WinObjC-Bold.ttf" />
+    <Font Include="..\..\CTCatalog\Fonts\WinObjC-Italic.ttf" />
+    <Font Include="..\..\CTCatalog\Fonts\WinObjC.ttf" />
```

Comment:
You seem to have added the fonts, but not committed to any testing strategy.

Can you add a visual/UI-driven test that renders text in the three winobjc fonts?

Do these fonts propagate to XAML controls? If so, can you set up a UIButton or UITextField that uses the winobjc font? That'd be cool.